packages/xo-server-audit/src/index.js

```diff
@@ -428,6 +438,46 @@ class AuditXoPlugin {
       throw error
     }
   }
+
+  async _removeRecord({ id }) {
```

Comment:
Why not allowing to remove a range of record?

Reply:
I didn't do it because the functionality was about removing one record, but i'll see if it's easy to extend it to a range of records.

Reply:
I think it's almost the same, therefore it would make sense to do it directly :-)